packages/website/sidebars/sidebar.base.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
           'getting-started/linting/type-linting',
           'getting-started/linting/monorepo',
           'getting-started/linting/troubleshooting',
+          'getting-started/linting/architecture',
```

Comment:
Question: I wonder if this belongs in a larger "about us" set of pages?

It's probs fine living here until we build out a few about pages.

Reply:
I dig it - maybe architecture, contributors, sponsors, financial support? 

Added a card to the [board](https://github.com/orgs/typescript-eslint/projects/1/views/1) named _"Flesh out "About Us" section in the site_.